bin/src/post-github-comment-from-travis.js

```diff
@@ -0,0 +1,44 @@
+import yargs from 'yargs';
+import { Transform } from 'stream';
+import { asCallback } from 'promise-callbacks';
+import postComment from '..';
+
+const { argv } = yargs
+  .usage('post-github-comment-from-travis [--purpose <purpose>]')
+  .alias('p', 'purpose')
+  .nargs('p', 1)
+  .describe('p', 'Set a comment purpose, to key on')
+  .epilog("pipe data into the command to set the comment's content")
```

Comment:
Neat!

Reply:
Kind of shame that yargs doesn't handle this for us. Maybe they'd be interested in that as a feature!

Reply:
Just discovered that the maintainer of `yargs` has a new CLI tool called [`sywac`](http://sywac.io/about/), which looks like it's more in this direction.

Reply:
whoa neat

Reply:
Also, how does no one ever talk about how there are as many CLI parsers as frontend JS frameworks

Reply:
heh I do all the time